locale/sv_SE/admin.xml

```diff
@@ -63,5 +78,21 @@
 	<message key="admin.mergeUsers.allUsers">Alla existerande användare</message>
 	<message key="admin.mergeUsers.confirm">Är du säker på att du vill lägga samman användarkontot: "{$oldUsername}" med: "{$newUsername}"? Användarkontot: "{$oldUsername}" kommer då att raderas. Åtgärden kan inte ångras.</message>
```

Comment:
This is no longer correct. The Swedish translation is rather outdated, and if your time permits it, a complete review probably would be very useful...